src/tests/build.proj

```diff
@@ -456,10 +456,6 @@
       <ExtraAppLinkerArgs Include="@(LinkerArg)" />
     </ItemGroup>
 
-    <PropertyGroup>
-      <NativeDependencies>$(IntermediateOutputPath)\..\$(TestRelativePath)\$(TestName)\native\$(TestName).o</NativeDependencies>
```

Comment:
Is this a fix for failing extra platforms pipeline?

Reply:
Hopefully yes. I didn't test it locally yet.